routes.go

```diff
@@ -122,6 +139,64 @@ func getTags(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 	w.Write(data)
 }
 
+func getBranchesfromFork(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
+	var err error
+
+	name := p.ByName("forkname")
+	if name == "" {
+		w.WriteHeader(http.StatusBadRequest)
+		fmt.Fprintf(w, "missing required parameter: name")
+		return
+	}
+
+	repo, err := git.PlainOpen(repoPath)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		fmt.Fprintf(w, "%v", err)
+		return
+	}
+	// Check if remote exists
+	remotes, err := repo.Remotes()
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		fmt.Fprintf(w, "%v", err)
+		return
+	}
+
+	remote_exists := false
```

Comment:
Another instance of a snake case variable name. Can you replace these with camel case?

```suggestion
	remoteExists := false
```